.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: php
+
+sudo: false
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+matrix:
+  include:
+    - php: 5.5
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+    - php: hhvm
+    - php: 7.0
+      env: COMPOSER_FLAGS="" SYMFONY_VERSION="3.0.*"
+
+before_install: if [[ "$SYMFONY_VERSION" != "" ]]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi
```

Comment:
We should disable the Xdebug extension if present and update Composer before doing everything else.